src/Telegram.Bot.Core/Exceptions/ApiExceptionParser.cs

```diff
@@ -5,9 +5,10 @@
 
 namespace Telegram.Bot.Exceptions
 {
-    internal static class ApiExceptionParser
+    public static class ApiExceptionParser
```

Comment:
It should stay internal. We had an idea of making exception parsing customizable, but we aren't ready to commit to this API.

Reply:
I made it public because it's placed in another assembly (`Telegram.Bot.Core`), but should be accessed from `Telegram.Bot`

So we have to decide what to do with this

Reply:
`InternalVisibleToAttribute` on assembly will do the job